src/middlewares/validation.js

```diff
@@ -272,14 +272,21 @@ const updateCartItem = [
     .withMessage('Quantity must be a number between 1 and 99.'),
 ];
 
-const ratingCheck = [
+const createReview = [
   check('rate')
     .notEmpty()
     .withMessage('rate cannot be empty.')
-    .isInt({ min: 1, max: 5 })
+    .isFloat({ min: 1, max: 5 })
```

Comment:
rate must be int so customer  can choose from int number and rating for chef can be float and can be int